### PR TITLE
Remove badge from atteendee endpoint

### PR DESCRIPTION
**Pull request by ARTurleite6**

### CODE REVIEW

File: lib/safira_web/controllers/deliver_prize_controller.ex [hunk 1]
```diff
@@ -43,6 +45,52 @@ defmodule SafiraWeb.DeliverPrizeController do
     end
   end
 
+  def delete(conn, %{"badge_id" => badge_id, "user_id" => user_id}) do
+    if Accounts.is_admin(conn) do
+      redeem = Contest.get_keys_redeem(user_id, badge_id)
+      referral = Contest.get_keys_referral(user_id, badge_id)
+      attendee = Accounts.get_attendee(user_id)
+      IO.inspect(redeem, label: "redeem")
```
Comment (MarioRodrigues10) [inline lib/safira_web/controllers/deliver_prize_controller.ex:53]: Please remove all ```IO.inspects()```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
And rename the PR to ```Remove badge from attendee endpoint```

File: lib/safira_web/controllers/deliver_redeemable_controller.ex [hunk 2]
```diff
@@ -5,6 +5,8 @@ defmodule SafiraWeb.DeliverRedeemableController do
 
   alias Safira.Store
 
+  alias Safira.Contest
```
Comment (MarioRodrigues10) [inline lib/safira_web/controllers/deliver_redeemable_controller.ex:8]: What is this alias used for?

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

File: lib/safira_web/controllers/deliver_redeemable_controller.ex (hunk 2 above)
Comment (ARTurleite6) [reply lib/safira_web/controllers/deliver_redeemable_controller.ex:8]: It was not needed actually, i forgot to remove it, sorry

**Review by MarioRodrigues10 — APPROVED** [review]
Good Job :+1:

**Pull request merged**